src/Submission/ReduxSubmissionHeader.js

```diff
@@ -25,7 +32,25 @@ const fetchUnmappedFileStats = user => dispatch => fetchWithCreds({
     }
   },
   err => ({ type: 'FETCH_ERROR', error: err }),
-).then((msg) => { dispatch(msg); });
+).then((msg) => {
+  if (msg) {
+    dispatch(msg);
+  }
+});
+
+const getStartingUUID = user => dispatch => fetchWithCreds({
+  path: `${indexdPath}index?acl=null&uploader=${user}&limit=1`,
+  method: 'GET',
+}).then(
+  ({ status, data }) => {
+    switch (status) {
+    case 200:
+      return data.records[0].did;
+    default:
+      return null;
+    }
+  },
+).then(res => dispatch(fetchUnmappedFileStats(user, [], res)));
```

Comment:
I noticed that this `getStartingUUID` and `fetchUnmappedFileStats` are similar to previous `getStartingUUID` and `fetchUnmappedFiles` in `ReduxMapFiles.js`, maybe some common code could moved into util files. But this is not urgent to this PR if we just want a quick fix MVP

Reply:
I agree it's tech debt, I'll try to move it out to a common function